ketcher-autotests/tests/User-Interface/Clear-Canvas/clear-canvas.spec.ts

```diff
@@ -0,0 +1,106 @@
+    await page.getByRole('dialog').getByRole('textbox').fill('12345');
+    await pressButton(page, 'Apply');
+    await selectTopPanelButton(TopPanelButton.Clear, page);
+    await takeEditorScreenshot(page);
+  });
+  test('Clear Canvas - Undo/Redo', async ({ page }) => {
+    // Test case: EPMLSOPKET-1704
+    await openFileAndAddToCanvas('reaction-dif-prop.rxn', page);
+    await clickInTheMiddleOfTheScreen(page);
+    await takeEditorScreenshot(page);
+    await selectTopPanelButton(TopPanelButton.Clear, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await selectTopPanelButton(TopPanelButton.Undo, page);
+    await selectTopPanelButton(TopPanelButton.Redo, page);
+    await selectTopPanelButton(TopPanelButton.Undo, page);
+    await takeEditorScreenshot(page);
+  });
+  test('Clear Canvas - Structure is opened from ket-file', async ({ page }) => {
+    // Test case:EPMLSOPKET-1705
+    await openFileAndAddToCanvas('ketcher.ket', page);
+    await clickInTheMiddleOfTheScreen(page);
+    await takeEditorScreenshot(page);
+    await selectTopPanelButton(TopPanelButton.Clear, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await selectTopPanelButton(TopPanelButton.Undo, page);
+    await page.keyboard.press('Control+Delete');
+    await takeEditorScreenshot(page);
+  });
+  test('Clear Canvas - Hotkeys', async ({ page }) => {
+    // Test case:EPMLSOPKET-2864
+    await addTextBoxToCanvas(page);
+    await clickInTheMiddleOfTheScreen(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('12345');
+    await pressButton(page, 'Apply');
+    await selectRing(RingButton.Benzene, page);
+    await page.getByTestId('canvas').click({ position: { x: 500, y: 250 } });
+    await delay(DELAY_IN_SECONDS.TWO);
+    await takeEditorScreenshot(page);
+    await page.keyboard.press('Control+Delete');
+    await openFileAndAddToCanvas('ketcher.mol', page);
+    await takeEditorScreenshot(page);
+    await page.keyboard.press('Control+Delete');
+    await takeEditorScreenshot(page);
+  });
+  test('Clear Canvas - Structure is opened from smile-string', async ({
+    page,
+  }) => {
+    // Test case: EPMLSOPKET-1706
+    await openFileAndAddToCanvas('chain-with-r-group.smi', page);
+    await clickInTheMiddleOfTheScreen(page);
+    await selectTopPanelButton(TopPanelButton.Clear, page);
+    await selectTopPanelButton(TopPanelButton.Undo, page);
+    await page.keyboard.press('Control+Delete');
+    await delay(DELAY_IN_SECONDS.TWO);
+    await selectTopPanelButton(TopPanelButton.Undo, page);
+    await selectTopPanelButton(TopPanelButton.Undo, page);
+    await selectTopPanelButton(TopPanelButton.Redo, page);
+    await selectTopPanelButton(TopPanelButton.Redo, page);
+    await takeEditorScreenshot(page);
+  });
+  test('Clear Canvas - Pressing Clear Canvas with Reaction Arrow under mouse cursor not causes errors in DevTool console', async ({
+    page,
+  }) => {
+    // await page.keyboard.press('Control+Shift+j');
```

Comment:
Why is it commented?
Also There is no devtools on screenshot

Reply:
Mikhail told me to leave this one, as we don’t yet have a function to track errors in the console